src/components/Nav/Navigation.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import logo from '../../assets/logo-rocky.png'
+import './StyleNavigation.css'
+
+const Navigation = (props) =>{
```

Comment:
es mejor destructurar
```suggestion
const Navigation = () =>{
```

Reply:
{ history }

Reply:
```suggestion
const Navigation = ({ history } ) =>{
```